vmm/src/config.rs

```diff
@@ -163,6 +163,10 @@ pub enum Error {
     /// Missing fields in Landlock rules
     #[error("Error parsing --landlock-rules: path/access field missing")]
     ParseLandlockMissingFields,
+    #[cfg(feature = "fw_cfg")]
+    #[error("Error parsing --fw_cfg_items")]
+    /// Failed Parsing FwCfgItem config
```

Comment:
Further, for all the other variants, `#[errors]` follows after the `/// doc comments`: please swap the lines to align style :)